lib/active_merchant/billing/gateways/paymentez.rb

```diff
@@ -9,7 +9,7 @@ class PaymentezGateway < Gateway #:nodoc:
 
       self.supported_countries = %w[MX EC CO BR CL PE]
       self.default_currency = 'USD'
-      self.supported_cardtypes = %i[visa master american_express diners_club elo alia olimpica]
+      self.supported_cardtypes = %i[visa master american_express diners_club elo alia olimpica discover maestro exito alkosto codensa sodexo olimpica epm carnet credisensa solidario union_pay jcb aura]
```

Comment:
Hey @ajawadmirza -- I think this list should only include card types that Active Merchant actually supports. That should be anything that's listed in CARD_COMPANY_DETECTORS here: https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/credit_card_methods.rb. Card types that aren't in that list will be rejected.

Reply:
Thanks, **Resolved**.